### DISCUSSION TITLE
allow selectDynamic to have optional type parameter - unified with type of field

### DISCUSSION THREAD

**Discussion by Katrix** [scala/scala3#18778] [Feature Requests]
Revived discussion on this I guess. https://github.com/lampepfl/dotty-feature-requests/issues/268

I thought this worked because the issue was closed and it seemed to be implemented, but it turns out it doesn't for some reason? Also can't find the commit that fixed it in the dotty commit graph. Not sure if I should open this as a bug or a feature request, but if I am correct in that the fix was never released, then I guess feature request?

**Comment by SethTisue** [comment]
I don't know why GitHub doesn't link to the new location the ticket was moved to: https://github.com/lampepfl/dotty/discussions/17818

(dotty-feature-requests repo was closed down and all the tickets became discussions in this repo)

**Comment by som-snytt** [comment]
It is also misleading that the feature requests were purple-closed for "completed". Purple-closing is not universally or uniformly respected in the repo anyway. But I'm already trained to salivate when I see purple.
